origin-contracts/contracts/token/TokenMigration.sol

```diff
@@ -67,5 +61,19 @@ contract TokenMigration is Ownable {
         emit MigrationFinished();
     }
 
+    // @dev Internal account migration function.
+    function _migrateAccount(address _holder) internal {
+        require(!migrated[_holder], "holder already migrated");
+        require(fromToken.paused(), "fromToken should be paused during migration");
```

Comment:
I think that checking `paused()` should only be done in public functions such as `migrateAccounts()` and `migrateAccount()`. Reads of contract state cost gas when performed during a transaction. So, I'd rather incur that cost once per call to `migrateAccounts()` instead of once per migrated account.

Reply:
You're right, that was the reason I wanted to create the new internal function in the first place. The requires indeed should be in the public function. I'll fix it later today:)